src/averageAges.js

```diff
@@ -34,7 +33,12 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  return people.filter(person =>
+    person.sex === 'f'
```

Comment:
```suggestion
  return people.filter(
  person => person.sex === 'f'
```
Не розриваємо коллбек. Якщо переносим на новий рядок, то всю начинку а не частину залишаємо